modules/rdma-configuring-the-sriov-operator.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * hardware_accelerators/rdma-remote-direct-memory-access.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="rdma-configuring-the-sriov-operator_{context}"]
+
+= Configuring the SR-IOV Operator
+
+Single root I/O virtualization (SR-IOV) enhances the performance of RDMA by providing sharing across multiple virtual machines from a single device. 
```

Comment:
"virtual machines or pods"?

Reply:
I would almost just say pods because a virtual machine in OCP is a pod technically.

Reply:
I agree, +1 to just say "pods".